nifi-marklogic-processors/src/main/java/org/apache/nifi/marklogic/processor/QueryRowsMarkLogic.java

```diff
@@ -65,23 +60,21 @@ public void init(ProcessorInitializationContext context) {
         Set<Relationship> set = new HashSet<>();
         set.add(FAILURE);
         set.add(SUCCESS);
-        set.add(ORIGINAL);
         relationships = Collections.unmodifiableSet(set);
     }
 
     @Override
     public void onTrigger(ProcessContext context, ProcessSessionFactory sessionFactory) throws ProcessException {
         final ProcessSession session = sessionFactory.createSession();
-        FlowFile incomingFlowFile = session.get();
-        if (incomingFlowFile == null) {
-            incomingFlowFile = session.create();
-        }
 
         try {
-            final String jsonPlan = determineJsonPlan(context, incomingFlowFile);
-            final String mimeType = determineMimeType(context, incomingFlowFile);
+            FlowFile flowFile = session.get();
```

Comment:
This looks like it's undoing the changes in QueryRowsMarkLogic - we want to preserve those and layer your changes on top.